src/api/exchange/ExchangeApiMock.ts

```diff
@@ -91,24 +88,23 @@ export class ExchangeApiMock extends DepositApiMock implements ExchangeApi {
       priceDenominator: orderParams.sellAmount,
       remainingAmount: orderParams.sellAmount,
     })
-    const orderId = this.orders[orderParams.userAddress].length - 1
 
-    return { data: orderId, receipt: RECEIPT }
+    return RECEIPT
```

Comment:
Well, the returned data is actually relevant I believe: https://github.com/gnosis/dex-contracts/blob/eb058c006e5ba929382c01851a25f3307e38c893/contracts/StablecoinConverter.sol#L179-L187

Or we'll just ignore it on the frontend?

Reply:
You don't get back `return` values from `solidity` contracts unless it's a `call` no?

Reply:
We can't get it on the frontend. 
When a contract calls a contract it does get the proper returned value.
When JS or anyone else but solidity calls a contract producing a change, that is transaction, what we get back is a transaction receipt.
If we need to then check the state after that transaction we either must make a call (not tx) to the contract or parse the tx receipt for events if there are any relevant to what data we need.

Reply:
Understood. Never mind then :)